README.md

```diff
@@ -11,7 +11,7 @@ Branches
 ------------
 These samples are released with Intel System Studio releases
 
-Latest Intel System Studio Release - master - Intel System Studio 2018 Update 1 (Current)
```

Comment:
@anjgola is this right?

Reply:
I updated the iss2019-update 1 branch. I think issdev-2045 needs to be updated.